src/main/java/com/microsoft/cse/helium/app/controllers/ActorsController.java

```diff
@@ -67,78 +59,28 @@
     } else {
       logger.error("Invalid actorId parameter " + actorId);
       return Mono.justOrEmpty(
-          ResponseEntity.badRequest().contentType(MediaType.TEXT_PLAIN)
-              .header("Invalid actorId parameter").build());
+          ResponseEntity.badRequest()
+              .contentType(MediaType.TEXT_PLAIN)
+              .header("Invalid actorId parameter")
```

Comment:
I don't think this error message is correct - run web validate to verify

Reply:
Done!